### PR TITLE
Clean up ApexPieChart.vue to not do additional type casting, simplify…

### PR DESCRIPTION
**Pull request by Bialogs**
… plot options. Fix issue with label inside the donut chart persisting by showing total.

### CODE REVIEW

**Review by rbclark — CHANGES_REQUESTED** [review]
The charts are currently failing to display on the deploy preview.

**Comment by Bialogs** [comment]
Blocked on https://github.com/apexcharts/apexcharts.js/issues/1923

**Comment by rbclark** [comment]
> Blocked on [apexcharts/apexcharts.js#1923](https://github.com/apexcharts/apexcharts.js/issues/1923)

More specifically, this is causing errors on the console whenever a user clicks on a donut slice due to the linked issue.

File: apps/frontend/src/components/generic/ApexPieChart.vue [hunk 1]
```diff
@@ -58,63 +32,21 @@ function next_id(): number {
...
-
-    // We now know the type is definitely number[]
-    let final = this.series as number[];
-
-    // If we have any non-zero data, just returngive 0.01 of all
-    return final;
-  }
+export default class ApexPieChart extends Vue {
+  @Prop({required: true}) readonly categories!: Category<string>[];
+  @Prop({required: true}) readonly series!: Number[];
```
Comment (rbclark) [inline apps/frontend/src/components/generic/ApexPieChart.vue:37]: Should be:

```typescript
  @Prop({required: true, type: Array}) readonly categories!: Category<string>[];
  @Prop({required: true, type: Array}) readonly series!: Number[];
```

**Comment by rbclark** [comment]
@Bialogs I skipped this in #342 since it was cleaned up here. Since it seems ApexCharts is not going to address our issue, I'm fine with us having the error display on the console since the actual functionality is improved even though there is an error.

**Pull request merged**